openmc/mgxs/library.py

```diff
@@ -13,6 +15,11 @@
 if sys.version_info[0] >= 3:
     basestring = str
 
+# The following represent the most accurate MGXS generation strategy
+# for use in the MG mode of OpenMC.
+OPENMC_MG_MGXS_TYPES = ['transport', 'absorption', 'nu-fission', 'chi',
```

Comment:
Perhaps include `'total'` here as an alternative to '`transport'`? (if and only if the scattering matrix does not use the P0 correction)